CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
-
+- Added support for "depends_on" and "depends_on_children" for export_specs
```

Comment:
This is an important and non-trivial feature.   I think the change log needs a few sentences to indicate at least when a user might want these.

Reply:
Ok, I will add these shortly

Reply:
Done